code-challenges.test.js

```diff
@@ -10,19 +11,22 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
-
-// b) Create the function that makes the test pass.
-
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
+// Pseudo Code: 
+// List declarations/variables
+// Create describe/it/expect statement for Jest test
+// List expected output(s) - whether or not the inputs are divisible by 3
+// Create function - divisibleBy3
+// Create (object) as parameter
+// Using object deconstruction, let {number} = object
+// Create if/else conditionals - if(number % 3 === 0){
+//     return `${number} is divisible by three`
+// }else if(number % 3 !== 0){
+//     return `${number} is not divisible by three`
+// }else {
+//     return "something went wrong"
+
 // a) Create a test with expect statements for each of the variables provided.
```

Comment:
Great pseudocode! Very detailed and specific, especially with the conditional statement.